### PR TITLE
chore: fix next build, update sdk, fix rechart

### PR DESCRIPTION
**Pull request by NamelsKing**
- fix nextjs build, https://github.com/Badger-Finance/badger-ninja/issues/22
- update sdk and utilize new types and data responses
- fix rechart lib, pinned to older version, (also was crashing nextjs app)
- provide correct txId for Yield historic links in explorers

### CODE REVIEW

File: components/VaultAprSources/index.tsx [hunk 1]
```diff
@@ -12,18 +12,29 @@ function toAprRange(apy: number, minApr?: number, maxApr?: number) {
...
+  const currentYieldDisplay = toAprRange(
+    apr.baseYield,
+    apr.minYield,
+    apr.maxYield,
+  );
 
   let yieldDisplay: React.ReactNode;
-  if (vault.sourcesApy.length > 0) {
-    yieldDisplay = vault.sources.map((s) => (
+  if (vault.apr.sources.length > 0) {
```
Comment (mrbasado) [inline components/VaultAprSources/index.tsx]: apy.sources

Comment (mrbasado) [reply components/VaultAprSources/index.tsx]: its in other spots too

Comment (NamelsKing) [reply components/VaultAprSources/index.tsx]: i was bit confused, coz comp called AprSources >_>

Comment (NamelsKing) [reply components/VaultAprSources/index.tsx]: but makes sense, fixed

**Review by mrbasado — APPROVED** [review]

**Pull request merged**